bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/mail/LicenseMailBuilder.java

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2019 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lic.api.mail;
```

Comment:
lic.api contains very basic concepts of passage, lets extract the email-related interfaces to lic.net

Also interface itself needs rework: "details" and "toText" should not depend on particular implementation, while the way to save as ".eml" should

I would expect something like 
```java
public void writeEml(MailDescriptor descriptor, OutputStream output) throws LicensingException
```